pkg/apis/kafka/v1alpha1/reset_offset_annotations_test.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestResetOffsetStatus_TopicAnnotation(t *testing.T) {
+	topic := "test-topic"
+	tests := []struct {
+		name  string
+		topic string
+	}{
+		{
+			name: "get non-existent",
+		},
+		{
+			name:  "set and get",
+			topic: topic,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			ros := &ResetOffsetStatus{}
+			if len(test.topic) > 0 {
+				ros.SetTopicAnnotation(test.topic)
+			}
+			got := ros.GetTopicAnnotation()
+			if diff := cmp.Diff(test.topic, got); diff != "" {
+				t.Errorf("unexpected conditions (-want, +got) = %v", diff)
+			}
```

Comment:
Yeah... I was just following the patterns in the existing api tests for consistency.  There is only one other use of testify in the apis/ that was subsequently added after the initial implementation.  The testify dependency shouldn't ever be a problem for anyone coding against the apis/ since they wouldn't pull in the _test.go files.

@matzew or @slinkydeveloper - Any opinions on using/not-using the testify assertions in the apis/ packages?  Concerns about consistency one way or another or just not a big deal either way?

Reply:
I'm +1000 on testify, manual assertions are just evil

Reply:
ok - good to hear - I agree - just a lot of the knative legacy tests are using -got +want and cmp.Diff, etc. so I wasn't sure.